rework description (#717)
deltachat/deltachat-android: store/text.md

# Short description (max. 80 characters)

Chat over e-mail. Simple and Secure. No silos, no tracking: Privacy by Design.

# Full description (max. 4000 characters)

> Delta Chat feels like Whatsapp or Telegram but does not involve centralized control

Reviewer: ```suggestion
Delta Chat looks and feels like other popular messenger apps, but does not involve centralized control
```

> or tracking. How does it achieve that? Delta Chat is technically an e-mail application 

Reviewer: ```suggestion
or tracking. Technically, Delta Chat is an e-mail application with a modern chat interface.
```
